Resources/Prototypes/Talos/Clothing/Cybersun/eyes.yml

```
@@ -1,9 +1,10 @@
 - type: entity
   parent: ClothingEyesGlassesChemical
   id: ClothingEyesCybersun
-  name: cargo tech jumpsuit
+  name: Защитные очки Cybersun
+  desc: Очки способные обеспечить защиту от попадания едких химикатов в глаза. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)